File: solutions/1769. Minimum Number of Operations to Move All Balls to Each Box/solution.cpp
class Solution {
public:
    vector<int> minOperations(string boxes) {
        vector<int> res;
        for(int i = 0; i < boxes.length(); ++i){
            int count = 0;
            for(int j = 0; j < boxes.length(); ++j){
                if(i != j){
                    if(boxes[j] == '1'){
                        count += abs(i-j);
                    }
                }
            }
            res.push_back(count);
        }
        return res;
    }
};

File: solutions/1769. Minimum Number of Operations to Move All Balls to Each Box/solution.py
class Solution(object):
    def minOperations(self, boxes):
        """
        :type boxes: str
        :rtype: List[int]
        """
        n = len(boxes)
        res = []
        for i in range(n):
            count = 0
            for j in range(n):
                if i != j:
                    if boxes[j] == '1':
                        count += abs(i-j)
            res.append(count)
        return res

        
